### DISCUSSION TITLE
Use XDG directories

### DISCUSSION THREAD

**Discussion by fdw** [cupcakearmy/autorestic#106] [Ideas]
As I see it, `autorestic` expects its configuration either in the current folder or in `$HOME`. I would like to add the canonical path for configuration files, `${XDG_CONFIG_HOME}` (by default `${HOME}/.config`), as this is a common standard.

**Comment by cupcakearmy** [comment]
Yes, might be a good idea :) Feel free to open an issue

**Reply by fdw** [reply]
Done: https://github.com/cupcakearmy/autorestic/issues/107
